ts/Series/Map/MapSeries.ts

```diff
@@ -682,6 +682,49 @@ class MapSeries extends ScatterSeries {
 
             // Add class names
             this.points.forEach((point): void => {
+                if (point.graphic) {
+
+                    const graphic = point.graphic,
+                        animate = point.graphic.animate;
+
+                    graphic.animate = function (params,
+                        options, complete): SVGElement {
+
+                        let switchBack = false;
+                        // When strokeWidth is animating
+                        if (params['stroke-width']) {
+                            const inheritedStrokeWidth = (
+                                (point.series.options.borderWidth || 0) /
```

Comment:
By hard coding this to `borderWidth`, we are missing the `lineWidth` option of the mapline (so it falls back to 0). Use the `pointAttrToOptions` approach that we use in `pointAttribs`.

And about that - since we're now doing that pattern three times in MapSeries, consider refactoring it out to a function. It can be local to the module if it works for both series types.